docs/toc.yml

```diff
@@ -7,6 +7,8 @@
 
 - name: Get started
   items:
+  - name: START HERE! A guide for beginners making Office Add-ins
```

Comment:
Caps seem weird to me, but maybe that is intentional?

Reply:
Agree, try either "Get started: A guide for beginners...." or omit.

Reply:
Kim suggested that the whole thing be shorter. As a compromise, let's try "Start Here! A guide for beginners"